src/tests/ftest/osa/online_drain.py

```diff
@@ -105,16 +106,17 @@ def run_online_drain_test(self, num_pool, oclass=None, app_name="ior"):
                     self.assert_on_exception()
 
         for val in range(0, num_pool):
+            self.pool = pool[val]
             display_string = "Pool{} space at the End".format(val)
-            self.pool[val].display_pool_daos_space(display_string)
-            self.run_ior_thread("Read", oclass, test_seq)
-            self.container = self.pool_cont_dict[self.pool[val]][0]
-            kwargs = {"pool": self.pool[val].uuid,
+            self.pool.display_pool_daos_space(display_string)
+            if app_name == "ior":
+                self.run_ior_thread("Read", oclass, test_seq)
+            self.container = self.pool_cont_dict[self.pool][0]
+            kwargs = {"pool": self.pool.uuid,
                       "cont": self.container.uuid}
             output = self.daos_command.container_check(**kwargs)
```

Comment:
pre-existing / minor - better to use `identifier`, which uses the label if existing. Also, minor, it's not necessary to use kwargs
```suggestion
            output = self.daos_command.container_check(pool=self.pool.identifier, cont=self.container.identifier)
```

Reply:
Done. Learnt a new thing :-)